src/app/parcels/add/formSections/CollectionDateCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { CardProps, onChangeDate, errorText } from "@/components/Form/formFunctions";
+import GenericFormCard from "@/components/Form/GenericFormCard";
+import StyledDatePicker from "@/components/DataInput/DatePicker";
+import { ErrorText } from "@/components/Form/formStyling";
+
+const CollectionDateCard: React.FC<CardProps> = ({
+    fieldSetter,
+    errorSetter,
+    formErrors,
+    fields,
+}) => {
+    return (
+        <GenericFormCard
+            title="Collection Date"
+            required={true}
+            text="What date is the client collecting their parcel?"
+        >
+            <>
+                <StyledDatePicker
+                    onChange={(value: any): void => {
+                        onChangeDate(fieldSetter, errorSetter, "collectionDate", value);
+                    }}
+                    label="Enter Date Here"
```

Comment:
Would it be better to call these "Date" "Time" instead of "Enter Date Here" and "Enter Time Here", mainly for being consistent with the `/client/add` page placeholders and it is also more concise!

Reply:
Changed